Reflections/R0 - April 9.md

```diff
@@ -0,0 +1,18 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+This week most of what I accomplished was Research into how game maker language works. I looked into it scalability as well and how it handles complex and Abstract data types. I also found a couple of YouTube channels that offer very helpful tutorials, namely Shaun Spalding and FriendlyCosmonaut. While I don't have a whole lot of code to show for this week, I can say that I have a much more confident understanding of how gml works and how best to use it.
```

Comment:
Research is work! After having looked at the syntax for GML, I'd definitely have to look at YoYo's documentation as well. They've created an interesting cross-paradigm language, that's for sure.